eng/pipelines/templates/stages/vmr-build.yml

```diff
@@ -144,6 +144,24 @@ stages:
       ### Additional jobs for full build ###
       - ${{ if in(parameters.scope, 'full') }}:
 
+        # This AlmaLinux leg is intended to build with the min supported glibc version
+        - template: ../jobs/vmr-build.yml
+          parameters:
+            # Changing the build name requires updating the referenced name in the source-build-sdk-diff-tests.yml pipeline
+            buildName: ${{ format('{0}_Online_MsftSdk', variables.almaLinuxName) }}
```

Comment:
This says "Online" but the parameter below sets `runOnline` to false. If the name is changed here, need to also update the sdk diff pipeline.

Reply:
Renamed to Offline since that was the intent.